assignments/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/assign.tagging.json

```diff
@@ -0,0 +1,31 @@
+{
+    "properties": {
+      "metadata": {
+        "assignedBy": "PlatOps"
+      },
+      "description": "Policy to check required tags",
+      "displayName": "HMCTS Tagging - CFT SBOX",
+      "enforcementMode": "DoNotEnforce",
+      "nonComplianceMessages": [
+        {
+            "message": "Resource doesn't meet the tagging requirements. See https://github.com/hmcts/azure-policy/blob/master/policies/tagging/README.md"
+        }
+      ],
+      "notScopes": [
+        "/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/resourceGroups/cft-sbox-00-aks-node-rg",
+        "/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/resourceGroups/cft-sbox-01-aks-node-rg",
+        "/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/resourceGroups/sbox-00-rg",
```

Comment:
typo on the resource groups, `cft-sbox` required

Reply:
sbox-00-rg is something old, this ignore is in relation to the aks resources that we deploy via https://github.com/hmcts/aks-cft-deploy - there was an issue when Blue implemented this with AKS and its tags. 

Do you want to test this without these exceptions to see if the tags/policy shows correctly?

Reply:
@thomast1906 - thanks, I'll add `cft-sbox-00-rg` to the list.

There is also a resource group `sbox-00-rg`. Are we not supposed to include https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/resourceGroups/sbox-00-rg/overview this one?

Reply:
@thomast1906 - I looked for `cft-sbox-00-rg` and 01 and didn't find any non-compliant resource, but I'll add the exemption for consistency.